release/release_tests.yaml

```diff
@@ -4524,8 +4549,13 @@
   working_dir: nightly_tests
 
   frequency: nightly
+  python: "3.8"
   team: core
   cluster:
+    byod:
+      type: cpu
```

Comment:
should we just make `type: cpu` the default type?

Reply:
and support `byod: ~`

Reply:
I like that!